Add curated extensions

carletex opened the pull request:
This PR allows you to call curated extensions as `-e <extension_name>` + doesn't show the "third-party extension" warning.

technophile-04 commented:
Ohh I see conflicts here, I think because we squash and merged #32  ? 

Also I think we should add changeset here so that we can release third party extension in next cli version 🙌

carletex commented:
> Ohh I see conflicts here, I think because we squash and merged https://github.com/scaffold-eth/create-eth/pull/32 ?

Yup, the squash & merged destroyed all the possible reconciliation haha. Fixed now!

Also added the changeset!
